src/IssueViz.Security/Hotspots/HotspotsStore.cs

```diff
@@ -46,10 +43,23 @@ internal interface IHotspotsStore
     [Export(typeof(IHotspotsStore))]
```

Comment:
I'm not quite satisfied with the MEF imports.
HotspotsStore is heavily aware of the inner implementation of IIssueVizsStore and it's underlying interfaces.

Reply:
Imports, or exports (there aren't any imports...).

What does the hotspots store know about the implementation of contained store that it shouldn't? It knows it implements a specific interface; anything else?

Having said that, it doesn't feel right. It looks like you are trying too hard to avoid using inheritance when that might be a cleaner implementation.